Events.md

```diff
@@ -1,10 +1,11 @@
 ## Past
 
-- October  29th 2015—[LLVM Developers' Meeting](http://llvm.org/devmtg/2015-10/slides/BastienGohman-WebAssembly-HereBeDragons.pdf), [Video](https://www.youtube.com/watch?v=5W7NkofUtAw).
-- December  9th 2015—[Emscripten and WebAssembly](https://kripken.github.io/talks/wasm.html)
-- January  31st 2016—[FOSDEM LLVM room](https://fosdem.org/2016/schedule/event/llvm_webassembly).
-- June     13th 2016-[NYLUG Presents: Luke Wagner -on- WebAssembly: A New Compiler](https://www.youtube.com/watch?v=RByPdCN1RQ4)
-- July      7th 2016-[Ben Titzer @ VMSS16: A Little on V8 and WebAssembly](https://www.youtube.com/watch?v=BRNxM8szTPA )
+- October   29th 2015—[LLVM Developers' Meeting](http://llvm.org/devmtg/2015-10/slides/BastienGohman-WebAssembly-HereBeDragons.pdf), [Video](https://www.youtube.com/watch?v=5W7NkofUtAw).
+- December   9th 2015—[Emscripten and WebAssembly](https://kripken.github.io/talks/wasm.html)
+- January   31st 2016—[FOSDEM LLVM room](https://fosdem.org/2016/schedule/event/llvm_webassembly).
+- June      13th 2016-[NYLUG Presents: Luke Wagner -on- WebAssembly: A New Compiler](https://www.youtube.com/watch?v=RByPdCN1RQ4)
```

Comment:
I think the title of this one got chopped off; it should end with "A New Compiler Target For The Web"  :)